govcd/api_vcd.go

```diff
@@ -102,6 +102,7 @@ func NewVCDClient(vcdEndpoint url.URL, insecure bool, options ...VCDClientOption
 					Proxy:               http.ProxyFromEnvironment,
 					TLSHandshakeTimeout: 120 * time.Second,
 				},
+				Timeout: 600 * time.Second,
```

Comment:
In the space of three lines we have three timeouts, all without comments. I think we should document all three of them.

Reply:
Added